CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [Unreleased]
+
+### Fixed
+- Fixed list_table create schema if user gives an nonexistent schema as params (#65) PR #63
```

Comment:
grammar issues
```suggestion
- Fixed `list_table` creating a schema if user gives a nonexistent schema passed as a query parameter. (#65) PR #63
```

Reply:
I reword this one to be  a bit more clear